dagql/idtui/viztest/main.go

```diff
@@ -431,3 +431,9 @@ func (*Viztest) List(ctx context.Context, dir *dagger.Directory) (string, error)
 	}
 	return strings.Join(ents, "\n"), nil
 }
+
+func (*Viztest) GitReadme(ctx context.Context, remote string) (string, error) {
+	result, err := dag.Git(remote).Head().Tree().File("README.md").Contents(ctx)
```

Comment:
not a blocker on merge here, but seems like this may get annoying to update whenever we change the README